src/domain/messages.ts

```diff
@@ -99,6 +105,7 @@ export type Message =
   | GameEndMessage
   | PlayerPositionsMessage
   | StampMessage
+  | FinishMessage
   | MapMessage
   | InvalidMessage
```

Comment:
nit: Was thining whether it would make sense to separate these to:
`MessageByServer` and `MessageByClient` to avoid mistakes.

```TypeScript
export type MessageByServer =
  | ErrorMessage
  | MapMessage
  | GameStartMessage
  | GameEndMessage
  | PlayerPositionsMessage
  | JoinMessage

export type MessageByClient = JoinMessage | MoveMessage | StampMessage | FinishMessage

export type Message = MessageByServer | MessageByClient
```

Reply:
👍